drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -336,6 +336,42 @@ def data_collection_features
     end.compact
   end
 
+  def occurrence_point_forms
```

Comment:
I think we will have duplication issue with this. Say for example there is a Form Definition for Move-in Date that has 4 rules (one enabling it for PH, one for RRH, etc.). We're looking at an Enrollment that has a `MoveInDate` _value_ but is not in a project where the Move-in Date form is _enabled_. Since we're iterating through all `instances` below, I think it will result in the Move In Date Form Definition being resolved 4 times on the Enrollment. 
Is that true?

Maybe we should tackle this more similarly to data_collection_features logic, aka check each latest-version of OCCURRENCE POINT definitions (by identifier) to see (1) is it enabled for this context, and if not (2) is there legacy data for it. Sorry not a fully formed suggestion, but lets make sure we are avoiding returning duplicates of the same form definition identifer